src/components/AElfWallet/AElfWallet.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * @file AElfWallet
+ * @author zhouminghui
+ * 最小屏适配可能会出现一些小问题 待查看
+ * 每次重新获取资产大概都需要把this.balance重置 直到我找到更好的解决办法。
+*/
+
+import React, {PureComponent} from 'react';
+import {Row, Col, Radio, Message, Spin} from 'antd';
+import Button from '../Button/Button';
+import Svg from '../Svg/Svg';
+import {tokenContract, aelf} from '../../utils';
+import formatNumber from '../../utils/formatNumber';
+import hexCharCodeToStr from '../../utils/hexCharCodeToStr';
+import getDividends from '../../utils/getDividends';
+import getConsensus from '../../utils/getConsensus';
+import getWallet from '../../utils/getWallet';
+import getStateJudgment from '../../utils/getStateJudgment';
+import './AElfWallet.less';
+
+const RadioGroup = Radio.Group;
+
+export default class AElfWallet extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.walletInfo = JSON.parse(localStorage.walletInfoList);
+        if (this.walletInfo != null) {
+            this.wallet = getWallet(JSON.parse(localStorage.currentWallet).privateKey);
+        }
+        this.consensus = getConsensus(this.wallet);
+        this.dividends = getDividends(this.wallet);
+        this.state = {
+            walletInfo: this.walletInfo,
+            value: JSON.parse(localStorage.currentWallet).privateKey,
+            refresh: 0,
+            loading: false,
+            loadingTip: null
+        };
+    }
+
+    async componentDidMount() {
+        await this.pushWalletInfo();
+    }
+
+    pushWalletInfo = async () => {
+        let walletInfo = this.state.walletInfo;
+        if (walletInfo != null) {
+            for (let i = 0; i < walletInfo.length; i++) {
```

Comment:
let i =0, j = walletInfo.length; i< j

Reply:
let walletInfo = xx || [];
walletInfo.map((item, index) => {})

Reply:
Has it been modified, but does map perform better than for loops?

Reply:
Has it been modified, but does map perform better than for loops?